src/app/lista-nominal/ListConteiner.tsx

```
@@ -0,0 +1,307 @@
+    useEffect(() => {
+        const fetchData = async () => {
+            setIsLoading(true);
+            setTableData({
+                data: [],
+                totalRows: 0,
+            });
+            setIsLoading(false);
+        };
+
+        if (session && session.user) fetchData();
+    }, [pagination, sorting, session, list]);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)